### PR TITLE
Context menu should only open when the map elements are selected

### PR DESCRIPTION
**Pull request by ivanaposdev**

### CODE REVIEW

File: src/app/components/map/events.ts [hunk 1]
```diff
@@ -6,12 +6,19 @@ import { actions } from '@components/map/actions';
 import { useAppContext } from '@contexts/app-context';
 
 const MARKER = 'Marker';
+const MAP = 'leaflet';
```
Comment (websterbontilao) [inline src/app/components/map/events.ts:9]: Why is the variable name called "MAP" and not "LEAFLET"?

File: src/app/components/map/events.ts [hunk 2]
```diff
@@ -6,12 +6,19 @@ import { actions } from '@components/map/actions';
...
+const STRING = 'string';
 
 export const Events = () => {
   const { handleShowDrawer } = useAppContext();
   const { handleContextMenuOpen, handleContextMenuClose, handleAction } =
     useMapContext();
 
+  const isMap = (e: L.LeafletMouseEvent) => {
+    const className = (e.originalEvent.target as HTMLInputElement).className;
+    return typeof className === STRING ? className.includes(MAP) : false;
```
Comment (websterbontilao) [inline src/app/components/map/events.ts:19]: can be shortened to 
`
return typeof className === STRING && className.includes(MAP);
`

**Review by websterbontilao — COMMENTED** [review]
Kindly see my comments and clarifications. TYIA

File: src/app/components/map/events.ts [hunk 3]
```diff
@@ -6,12 +6,19 @@ import { actions } from '@components/map/actions';
...
 const MARKER = 'Marker';
+const MAP = 'leaflet';
+const STRING = 'string';
 
 export const Events = () => {
   const { handleShowDrawer } = useAppContext();
   const { handleContextMenuOpen, handleContextMenuClose, handleAction } =
     useMapContext();
 
+  const isMap = (e: L.LeafletMouseEvent) => {
```
Comment (websterbontilao) [inline src/app/components/map/events.ts:17]: You can also add boolean return type:
  
`
const isMap = (e: L.LeafletMouseEvent): boolean => {
   ...
}
`

File: src/app/components/map/events.ts (hunk 1 above)
Comment (ivanaposdev) [reply src/app/components/map/events.ts]: Changed `MAP` to `LEAFLET`

File: src/app/components/map/events.ts (hunk 2 above)
Comment (ivanaposdev) [reply src/app/components/map/events.ts]: Good suggestion, done changing. 🚀

**Review by websterbontilao — APPROVED** [review]

**Pull request merged**